FinSearchUnified/Tests/Bundle/ProductNumberSearchTest.php

```diff
@@ -21,11 +25,11 @@ protected function setUp()
 
         $configArray = [
             ['ActivateFindologic', true],
-            ['ShopKey', 'ABCD0815'],
+            ['ShopKey', 'ABCDABCDABCDABCDABCDABCDABCDABCD'],
```

Comment:
Thanks for using realistic shopkeys :)

Reply:
Is there any reason why we write it `ShopKey` instead of just `shopkey`?

Reply:
Yes that's actually what's used in the plugin config unfortunately.